src/pspm_load_data.m

```diff
@@ -232,17 +228,23 @@
     clear loaded_data
   end
 end
-% initialise error flags
+%% 7 Check data
+% 7.1 initialise error flags --
 vflag = zeros(numel(data), 1); % records data structure, valid if 0
 wflag = zeros(numel(data), 1); % records whether data is out of range, valid if 0
 nflag = zeros(numel(data), 1);
 zflag = zeros(numel(data), 1); % records whether data is empty
 % loop through channels
 for k = 1:numel(data)
-  % check header
+  % 7.2 Check header --
   if ~isfield(data{k}, 'header')
     vflag(k) = 1;
   else
+    % 7.2.1 Convert header channeltype into chantype if there are --
```

Comment:
The code of converting `channeltype` into `chantype` is moved to this place because this loop assumes that there is a field `chantype` there for checking.